src/main/java/mekanism/common/util/GasUtils.java

```diff
@@ -27,13 +27,10 @@
 public final class GasUtils {
 
     public static IGasHandler[] getConnectedAcceptors(BlockPos pos, World world, Set<EnumFacing> sides) {
-        IGasHandler[] acceptors = new IGasHandler[]{null, null, null, null, null, null};
-        for (EnumFacing orientation : sides) {
-            TileEntity acceptor = world.getTileEntity(pos.offset(orientation));
-            if (CapabilityUtils.hasCapability(acceptor, Capabilities.GAS_HANDLER_CAPABILITY, orientation.getOpposite())) {
-                acceptors[orientation.ordinal()] = CapabilityUtils.getCapability(acceptor, Capabilities.GAS_HANDLER_CAPABILITY, orientation.getOpposite());
-            }
-        }
+        final IGasHandler[] acceptors = new IGasHandler[]{null, null, null, null, null, null};
+        EmitUtils.forEachSide(world, pos, sides, (tile, side) -> {
+            acceptors[side.ordinal()] = CapabilityUtils.get(tile, Capabilities.GAS_HANDLER_CAPABILITY, side.getOpposite());
```

Comment:
Shouldn't this be `getCapability` instead of `get`?

Reply:
Yes, looks like my local failed to do git.... its been a problem with gitkaken recently.